src/Commands/Component.php

```diff
@@ -0,0 +1,12 @@
+<?php
+namespace Airwire\Commands;
+
+use Illuminate\Support\Str;
+
+class Component {
```

Comment:
There are a few issues with this class:
- It's essentially a helper class, which isn't a good practice. Methods should be on objects, like `$plant->water()` rather than `PlantWaterer::water($plant)`. Adam has a great talk about this: https://www.youtube.com/watch?v=dfgtKb-VpRk
- It's in the `Airwire\Commands` namespace, where each class should be a command
- Code style should match the rest of the project, i.e. `{` on new line. I'll add PHP CS Fixer to automatically resolve these issues